packages/router/src/config.ts

```diff
@@ -63,6 +71,11 @@ export const TChainConfig = Type.Object({
   subgraph: Type.String(),
   transactionManagerAddress: Type.String(),
   minGas: Type.String(),
+  feePercentage: Type.Number({ minimum: minimumFeeAllowed, maximum: maximumFeeAllowed }),
+  safeRelayerFeePercentage: Type.Number({
```

Comment:
This should not be in here. The relayer fee will not be a percentage. It will simply be the gas fee + some additional cost (probably in USD).